docs/navigation-parameters-reference.md

```diff
@@ -96,6 +96,12 @@ The navigation parameters allow you to configure **global** navigation settings
 - **description**: renders all views in new frames. This setting overrides the same-domain frame reuse.
 - **default**: the parameter **defaults.isolateView** is `false` by default, and you can overwrite it using the **isolateView** value on a single node level.
 
+### tooltipText
+- **type**: string
+- **description**: Add tooltip to all of the links and buttons of the left side navigation and top navigation.
+- **default**: the parameter **defaults.tooltipText** is `null` by default, and you can overwrite it using the **tooltipText** value on a single node level.
+
```

Comment:
I'm not sure if `defaults.tooltipText` is a separate parameter, for example like [isolateView](https://docs.luigi-project.io/docs/navigation-advanced/?section=isolateview) and [deafults.isolateView](https://docs.luigi-project.io/docs/navigation-advanced/?section=defaultsisolateview). In that case, I think it would be better to write a separate description for it 🙂 But I'm not sure what that description should be/what it does, @ndricimrr what do you think? 
For example, 
```
### defaults.tooltipText
- **type**: string
- **description**: <!-- ??? sets the default tooltip text for every navigation node ???  --> 
- **default**: the parameter **defaults.tooltipText** is `null` by default, and you can overwrite it using the **tooltipText** value on a single node level.
```

Reply:
Ok but now we have `Add tooltip to all of the links and buttons of the left side navigation and top navigation.`  so you suggest  `sets the default tooltip text for every navigation node`? @aleksandr-suvorov

Reply:
Right, the question is if `tooltipText` adds it to *all* the navigation nodes or just one? 
As far as I understand, `defaults.tooltipText` adds a tooltip to all the nodes, and `tooltipText` to individual nodes.
Thanks, I changed my suggestion above to reflect this.

Reply:
But `defaults.tooltipText` will not have sense. The idea was that the client could add any tooltip himself, e.g. when the length of the label exceeds the width of the button / navigation.